src/pages/contact-us/_need-help.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Text } from '../../components/elements/typography'
+import { localize, LocalizedLink } from 'components/localization'
+import { Header } from 'components/elements'
+import NeedUsIcon from 'images/svg/need-us.svg'
+
+const Wrapper = styled.section`
+    width: 100%;
+    height: 35rem;
+    background-color: var(--color-grey-8);
+    padding: 4rem 6.6rem;
+`
+
+const SectionWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 100%;
+`
+const Logo = styled(NeedUsIcon)`
+    margin-bottom: 1.4rem;
+`
+
+const StyledHeader = styled(Header)`
+    margin-bottom: 0.8rem;
+`
+const StyledText = styled(Text)`
+    margin-bottom: 2.4rem;
+    width: 38.4rem;
+`
+const StyledButton = styled(LocalizedLink)`
```

Comment:
is it the same as button that we have?, if it is, please use Button component instead and use navigate, or else we can have new shared button link component

Reply:
Actually I had a task before (404 page), and Oskar asked me to create a button based on link, but I was supposed to make it a shared component because it is used in other pages too. I will do that asap, on all pages we have used so far.